accounts/models.py

```diff
@@ -351,3 +354,37 @@ def get_reputation_history(self):
             'labels': labels,
             'points': points
         }
+
+    def get_reputation_change(self, date):
+        """
+        Get change of reputation since date
+        :return: change of reputation
+        :rtype: int
+        """
+        snapshots = self.snapshots.filter(
+            snapshot_of_id=self.id,
+            created_at=date,
+        ).order_by('created_at')
+        if len(snapshots):
+            old_reputation = self.reputation_formula(snapshot.portfolio_value,
```

Comment:
Uh, need more tests
9 kwi 2016 00:05 "Wojciech Zając" <notifications@github.com> napisał(a):

> In accounts/models.py
> <https://github.com/KlubJagiellonski/Politikon/pull/129#discussion_r59095660>
> :
>
> > @@ -351,3 +354,37 @@ def get_reputation_history(self):
> >              'labels': labels,
> >              'points': points
> >          }
> > +
> > +    def get_reputation_change(self, date):
> > +        """
> > +        Get change of reputation since date
> > +        :return: change of reputation
> > +        :rtype: int
> > +        """
> > +        snapshots = self.snapshots.filter(
> > +            snapshot_of_id=self.id,
> > +            created_at=date,
> > +        ).order_by('created_at')
> > +        if len(snapshots):
> > +            old_reputation = self.reputation_formula(snapshot.portfolio_value,
>
> snapshots vs snapshot. Typo?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/KlubJagiellonski/Politikon/pull/129/files/065691040ee7865eeff1444a0cd565ab859e660a#r59095660>
>